### ISSUE TITLE
Cannot run program "dart_format": error=2, No such file or directory

### ISSUE THREAD

**Issue by smitKmsoft** [eggnstone/DartFormatJetbrainsPlugin#7]
Please supply any additional information here, e.g. the source code that caused the error:

```
java.io.IOException: Cannot run program "dart_format": error=2, No such file or directory
	at java.base/java.lang.ProcessBuilder.start(Unknown Source)
	at java.base/java.lang.ProcessBuilder.start(Unknown Source)
	at dev.eggnstone.plugins.jetbrains.dartformat.plugin.ExternalDartFormat$run$process$1.invokeSuspend(ExternalDartFormat.kt:120
```
Origin: ExternalDartFormat.run/1

**Comment by eggnstone** [comment]
Please install the dart_format package:
Please look here: https://pub.dev/packages/dart_format/install

Does the error still occur?

Merging with existing issue ...

**Comment by eggnstone** [comment]
Merged with #5

**Comment by eggnstone** [comment]
@smitKmsoft 
Please check the latest version (2.5) of the DartFormat plugin:
https://plugins.jetbrains.com/plugin/21003-dartformat/versions
This one should fix the problem or at least give better diagnostics.
